code/modules/reagents/chemistry/machinery/smoke_machine.dm

```diff
@@ -11,9 +11,9 @@
 	var/cooldown = 0
 	var/screen = "home"
 	var/useramount = 30 // Last used amount
-	var/volume = 1000
+	var/volume = 300
```

Comment:
stealth nerf

Reply:
🚶

Reply:
no put this in the pr description, not tolerated

Reply:
^^^^^^^^^^^^^^^^^^^^^

Reply:
🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶